### PR TITLE
Remove deprecated setting historical_data_start

### LINKED ISSUE #2017: Remove unused setting historical_data_start
## Problem Definition

With the current work I am doing in the cryptocompare query the `historical_data_start` setting is no longer needed. It was only used as a way to know how far back to query for prices of each asset to save time. Now that the cryptocompare histohour queries are moving to a background task we will just get all available data.

## Task

Remove the setting.

### PR DESCRIPTION
**Pull request by LefterisJP**
Fix #2017

### CODE REVIEW

File: frontend/app/src/views/settings/GeneralSettings.vue [hunk 1]
```diff
@@ -27,39 +27,6 @@
...
-              offset-y
-              max-width="290px"
-              min-width="290px"
-            >
-              <template #activator="{ on }">
-                <v-text-field
-                  v-model="historicDataStart"
-                  class="general-settings__fields__historic-data-start"
-                  :label="$t('general_settings.labels.historical_data_start')"
-                  :hint="$t('general_settings.historic_start_hint')"
```
Comment (kelsos) [inline frontend/app/src/views/settings/GeneralSettings.vue:45]: also the hint has to be removed from the en.json

File: frontend/app/src/views/settings/GeneralSettings.vue [hunk 2]
```diff
@@ -650,40 +604,6 @@ export default class General extends Settings {
...
-    if (!this.notTheSame(date, previousValue)) {
-      return;
-    }
-
-    const params = {
-      date
-    };
-    const message: BaseMessage = {
-      success: `${this.$t(
-        'general_settings.validation.historic_data.success',
```
Comment (kelsos) [inline frontend/app/src/views/settings/GeneralSettings.vue:667]: This key also need to be removed from en.json

**Review by kelsos — APPROVED** [review]
Other than the 3 settings we discussed on discord, if the frontend tasks pass we are good

File: frontend/app/src/views/settings/GeneralSettings.vue (hunk 2 above)
Comment (LefterisJP) [reply frontend/app/src/views/settings/GeneralSettings.vue:667]: done

**Comment by LefterisJP** [comment]
Finally merging an approved PR

**Pull request merged**